website/docs/pipelines/promoting-applications.mdx

```diff
@@ -140,118 +142,170 @@ is upgraded in one of the pipeline's environments, a PR is created that updates
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- policies/RBACProhibitWildcards/no-wildcard-on-resources-policy.yaml
+- policies/RBACProhibitWildcards/no-wildcard-on-verbs-policy.yaml
+- policies/RBACProhibitVerbsOnResources/no-list-secrets-policy.yaml
+- policies/RBACProhibitVerbsOnResources/no-watch-secrets-policy.yaml
+- policies/RBACProhibitVerbsOnResources/no-wildcard-secrets-policy.yaml
+```
 
-:::tip
+An example of policy to reject using wildcards (in this case using wildcards on verbs) is shown below:
 
-Adopt as much of the recommendations in this section to reduce the risks associated with the secrets involved in pull requests.
+```yaml
+apiVersion: pac.weave.works/v2beta2
+kind: Policy
+metadata:
+  name: weave.templates.rbac-prohibit-wildcards-verbs
+spec:
+  id: weave.templates.rbac-prohibit-wildcards-verbs
+  name: Rbac Prohibit Wildcards on Verbs
+  ...
+  category: weave.categories.access-control
+  severity: high
+  parameters:
+    - name: attributes
+      type: string
+      required: true
+      value: "verbs"
+    - name: exclude_role_name
+      type: string
+      required: true
+      value: "wego-admin"
+    - name: exclude_label_key
+      type: string
+      required: false
+      value: "app.kubernetes.io/part-of"
+    - name: exclude_label_value
+      type: string
+      required: false
+      value: "flux"
+```
 
-:::
+An example of policy to reject allowing list secrets is shown below:
 
-1. **Create a user account for pull request changes**: this user context would be used to do any git provider operation,
-and for security and auditing perspective, you don't want to impersonate a real user for it.
+```yaml
+apiVersion: pac.weave.works/v2beta2
+kind: Policy
+metadata:
+  name: weave.templates.rbac-prohibit-list-on-secrets
+spec:
+  id: weave.templates.rbac-prohibit-list-on-secrets
+  name: Rbac Prohibit Wildcard On Secrets
+  ...
+  category: weave.categories.access-control
+  severity: high
+  parameters:
+    - name: resource
+      type: string
+      required: true
+      value: "secrets"
+    - name: verb
+      type: string
+      required: true
+      value: "list"
+    - name: exclude_label_key
+      type: string
+      required: false
+      value: "app.kubernetes.io/part-of"
+    - name: exclude_label_value
+      type: string
+      required: false
+      value: "flux"
+```
 
-<details><summary>Expand to see example</summary>
+An example of privilege escalation for pipelines blocked via Policy Agent is shown below
 
-![using bot account](img/bot-account.png)
-</details>
+![privilege escalation blocked](img/pipeline-priviledge-escalation-blocked.png)
 
-2. **Restrict access to the secret**: the promotion credentials needs to reside in the same Namespace as the Pipeline resource on the management cluster. Restrict
-via RBAC that only `pipeline-controller` service account is able to read it.
 
-<details><summary>Expand to see example</summary>
+#### Service Account
```

Comment:
controller should live in its namespace

Reply:
This concern it is reflected in the shared security context board. it will be part of the future security documentation.